chart/templates/stub-connector-deployment.yaml

```diff
@@ -25,6 +23,8 @@ spec:
         app.kubernetes.io/name: connector
         app.kubernetes.io/instance: {{ .Release.Name }}
         talksToHsm: "true"
```

Comment:
I'm going to remove this is a separate PR, the stub-connector no longer talks to the HSM